wiki/People/osu!_team/en.md

```diff
@@ -11,29 +11,39 @@ tags:
+| ::{ flag=CA }:: [Azer](https://osu.ppy.sh/users/2155578) | [World Cup](/wiki/Tournaments#official-world-cups) organiser and administrator |
 | ::{ flag=US }:: [ChillierPear](https://osu.ppy.sh/users/9501251) | World Cup organiser and administrator |
-| ::{ flag=AU }:: [Ephemeral](https://osu.ppy.sh/users/102335) | Project and community maintainer, Featured Artist outreach |
+| ::{ flag=US }:: [Death](https://osu.ppy.sh/users/3242450) | [Moderator](/wiki/People/Global_Moderation_Team), technical support |
+| ::{ flag=US }:: [Dntm8kmeeatu](https://osu.ppy.sh/users/5428812) | Moderator, technical support |
+| ::{ flag=SK }:: [Domco](https://osu.ppy.sh/users/3562660) | osu! developer |
+| ::{ flag=AU }:: [Ephemeral](https://osu.ppy.sh/users/102335) | Project and community maintainer, [Featured Artist](/wiki/People/Featured_Artists) outreach |
 | ::{ flag=JP }:: [flyte](https://osu.ppy.sh/users/3103765) | Design lead, resident Creative Cloud watcher |
-| ::{ flag=TN }:: [Hivie](https://osu.ppy.sh/users/14102976) | Modding and mapping scene advisor |
+| ::{ flag=SA }:: [frenzibyte](https://osu.ppy.sh/users/14210502) | osu! developer |
+| ::{ flag=TN }:: [Hivie](https://osu.ppy.sh/users/14102976) | [Nomination Assessment Team](/wiki/People/Nomination_Assessment_Team) leader, [Beatmap Spotlights](/wiki/Beatmap_Spotlights) manager, radar's secretary |
 | ::{ flag=BR }:: [LeoFLT](https://osu.ppy.sh/users/3668779) | World Cup organiser and administrator |
-| ::{ flag=GB }:: [mangomizer](https://osu.ppy.sh/users/1893718) | Tournament support, Featured Artist outreach |
+| ::{ flag=GB }:: [mangomizer](https://osu.ppy.sh/users/1893718) | Featured Artist outreach, World Cup secretary |
+| ::{ flag=NO }:: [MillhioreF](https://osu.ppy.sh/users/941094) | osu! developer, technical support |
 | ::{ flag=JP }:: [nanaya](https://osu.ppy.sh/users/2387883) | osu!web developer |
 | ::{ flag=PH }:: [Nathanael](https://osu.ppy.sh/users/2295078) | General housekeeper, moderation advisor |
 | ::{ flag=SE }:: [Naxess](https://osu.ppy.sh/users/8129817) | Modding and mapping scene advisor |
-| ::{ flag=AU }:: [nekodex](https://osu.ppy.sh/users/102) | osu!web developer, [official osu! Featured Artist](https://osu.ppy.sh/beatmaps/artists/1) |
+| ::{ flag=AU }:: [nekodex](https://osu.ppy.sh/users/102) | osu!web developer, [Featured Artist](https://osu.ppy.sh/beatmaps/artists/1) |
 | ::{ flag=JP }:: [notbakaneko](https://osu.ppy.sh/users/10751776) | osu!web developer |
-| ::{ flag=US }:: [pishifat](https://osu.ppy.sh/users/3178418) | Modding and mapping scene advisor, Mappers' Guild organiser, Featured Artist outreach |
-| ::{ flag=US }:: [radar](https://osu.ppy.sh/users/7131099) | Modding and mapping scene advisor |
-| ::{ flag=MX }:: [Repflez](https://osu.ppy.sh/users/201392) | General housekeeper, wiki maintainer |
+| ::{ flag=DE }:: [Okoratu](https://osu.ppy.sh/users/1623405) | Moderator |
```

Comment:
Reducing the role of the moderators listed here vs. the moderators listed in the GMT directory does nothing and will continuously add confusion. It is a bit reductive for nothing IMHO.

Reply:
just gmt is more accurate now than what it was previously when they were listed as inactive

Reply:
Idk, I guess being gmt = osu! team, so then the GMT directory is just every GMT vs moving them over to this directory?

Reply:
i've added back what was there before